WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -480,7 +480,12 @@ class StoryComposerActivity : ComposeLoopFrameActivity(),
             val blockId = intent.extras?.getString(ARG_STORY_BLOCK_ID)
             savedContentIntent.putExtra(ARG_STORY_BLOCK_ID, blockId)
 
-            val storyIndex = intent.getIntExtra(KEY_STORY_INDEX, DEFAULT_NONE_SELECTED)
+            // check if story index has been passed through intent
+            var storyIndex = intent.getIntExtra(KEY_STORY_INDEX, DEFAULT_NONE_SELECTED)
+            if (storyIndex == DEFAULT_NONE_SELECTED) {
+                // if not, let's use the current Story
+                storyIndex = storyRepositoryWrapper.getCurrentStoryIndex()
```

Comment:
@mzorz is it redundant to set this since it's `DEFAULT_NONE_SELECTED ` as well? Let me know if I am missing something. Thanks 🙏

Reply:
Good catch! it actually is not redundant, because it may or may be not `DEFAULT_NONE_SELECTED`. (it may happen the value _is actually_ set in extras as -1, for example). Si yeah we need to check for it 👍

Reply:
Okay understood! I get what you are saying now because the currentStoryIndex is definitely not always the default as it starts out since it's within a singleton. thanks for expounding 🙏